pkg/volume/gcepd/gce_util.go

```diff
@@ -356,7 +356,7 @@ func udevadmChangeToDrive(drivePath string) error {
 // Checks whether the given GCE PD volume spec is associated with a regional PD.
 func isRegionalPD(spec *volume.Spec) bool {
 	if spec.PersistentVolume != nil {
-		zonesLabel := spec.PersistentVolume.Labels[v1.LabelFailureDomainBetaZone]
+		zonesLabel := spec.PersistentVolume.Labels[v1.LabelTopologyZone]
```

Comment:
According to what I found, this will only be called in the in-tree gcepd Attach() function. Hmm, yes it might also need to add beta label support in case of version skew for kubelet and master components.